MetroscopeModelingLibrary/MultiFluid/HeatExchangers/FuelHeater.mo

```diff
@@ -126,7 +126,6 @@ equation
   DT_hot_in_side = T_hot_in - T_cold_out;
   DT_hot_out_side = T_hot_out - T_cold_in;
   pinch = min(DT_hot_in_side, DT_hot_out_side);
-
   assert(pinch > 0, "A very low or negative pinch is reached", AssertionLevel.warning); // Ensure a positive pinch
```

Comment:
Wouldn't a zero pinch fail the simulation anyway? Not sure this assert is relevant. Maybe flagging low pinches (<1°C ?) would be more interesting.

Reply:
It simulates, I tested it. Because sometimes the energy balance is correct if we have sufficient flow.

Reply:
agreed !

Reply:
Maybe in this case let's add a second assert at the info level, for low pinches

Reply:
I didn't understand the. For flue gases heat exchangers, there's an assert for W < 0, do you mean something similar ?

Reply:
No, I meant an assert for low pinches (not zero pinches), like pinch < 1°C